api/institutions/serializers.py

```diff
@@ -191,3 +191,39 @@ def get_absolute_url(self, obj):
                 'version': 'v2',
             },
         )
+
+
+class InstitutionUserMetricsSerializer(JSONAPISerializer):
+
+    class Meta:
+        type_ = 'institution-users'
+
+    id = IDField(read_only=True)
+    user_id = IDField(read_only=True)
+    user = ser.CharField(read_only=True)
```

Comment:
Because I think the relationship for user should be called `user` (see below), this should probably be `user_name`.